internal/c99/ast2.go

```diff
@@ -195,6 +195,14 @@ func (d *DeclarationSpecifier) typ(ctx *context) Type {
 		return UShort
 	case d.is(TypeSpecifierShort, TypeSpecifierSigned):
 		return Short
+	// TODO: not sure if this is right
```

Comment:
It does not look correct to me. It probably indicates a bug in the frontend. Can you please provide a small example that triggers this case? You can put it right into c99/testdata/bug/nn.c, thanks.

Reply:
What's the easiest way to track this down to the original file location?
Doing something like fmt.Println(d.TypeSpecifiers[0].Pos()) only gives me the position of the token and not the file.

Reply:
I hope `ctx.position(d)` should provide that.

Reply:
No, `DeclarationSpecifier` is not a `Node`. But any non nil item within the `*FunctionSpecifier`, ... slices is a `Node`. I should add the `.Pos()` method to `DeclarationSpecifier`.

Reply:
It seems to work with ctx.position(d.TypeSpecifiers[0]))

Reply:
Reproduction in https://github.com/cznic/sqlite2go/pull/7/commits/27c997917dab8ef7e3a765bb6a19ec54782afea7
(testdata/bugs/20.c)